templates/_seo_extension_field.twig

```diff
@@ -49,6 +49,15 @@
     {% set seovalues = seovalues|merge({'robots': seoconfig.meta_robots|default("index, follow")})  %}
 {% endif %}
 
+{# show/hide fields #}
+{% set show = {
+    shortlink : seoAllowed('shortlink'),
+    canonical : seoAllowed('canonical'),
+    robots    : seoAllowed('robots'),
+    ogtype    : seoAllowed('ogtype')
+} %}
```

Comment:
It is a fairly tiny difference, but wouldn't it be better/faster to have `seoAllowed()` return an array rather than always making 4 calls?

Reply:
Good call!